web/templatetags/membership_tags.py

```
@@ -0,0 +1,27 @@
+
+
+@register.simple_tag
+def is_active_membership(user):
+    """Check if a user has an active membership."""
+    if hasattr(user, "membership"):
+        return user.membership.is_active
+    return False
+
+
+@register.simple_tag
+def get_membership_plan_name(user):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_membership_plan_name`

(ANN201)